src/logic/calculate.js

```diff
@@ -95,6 +99,13 @@ export default function calculate(obj, buttonName) {
     return {};
   }
 
+  // no operation yet, but the user typed one
+
+  // The user hasn't typed a number yet, just save the operation
+  if (!obj.next) {
```

Comment:
* I get this error when i perform division by zero operation 
![image](https://user-images.githubusercontent.com/52497246/129296807-72e8348d-3e3e-4c56-b31a-3b82b7759fb6.png)